src/main/java/com/cburch/logisim/fpga/hdlgenerator/ToplevelHdlGeneratorFactory.java

```diff
@@ -232,4 +234,61 @@ public boolean hasLedArrayType(String type) {
     }
     return contents.get();
   }
+
+  private static ArrayList<String> getToplevelCode(MapComponent component) {
+    final var temp = new StringBuffer();
+    final var contents = new ArrayList<String>();
+    if (component.getNrOfPins() <= 0) {
+      Reporter.report.addError("BUG: Found a component with no pins");
```

Comment:
If this is a `bug` and not regular message, the chance it occurs are low. Wouldn't it be better then to also add something like `Please report this occurence.` call to the message?

Reply:
Done